UI/src/view/csvReportsWorkList.js

```diff
@@ -0,0 +1,489 @@
+(function (root) {
+  root.csvReportsWorkList = function () {
+    WorkListView.call(this);
+    var me = this;
+    this.hrefDir = "#work-list/csvReports";
+    this.title = 'Raportointityökalu';
+    var backend;
+    var municipalities;
+    var assetsList;
+    var refresh;
+
```

Comment:
We can build a structure here, something immutable, to have an easy access to the id and value from assets like this, feel free to change the names
``` JavaScript
var options = {
     assets: {id: "1", value: "Kaikki tietolajit"}, 
     points: {id: "2", value: "Pistemäiset tietolajit"}, 
     linears: {id: "3", value: "Viivamaiset tietolajit"}
}
```
Here we also have the option to add ```municipalities: {id: "1000", value: "Kaikki kunnat"} ``` and distinguish it with a boolean value in the same structure or just build it separate from the others. 
This structure can now be accessed like this 
``` JavaScript
options["points"].id
```
We can use this structure in the code when needed. I will be pointing where this can be used in the comments

Reply:
apllied.